app/models/project_metadata.rb

```diff
@@ -130,39 +125,60 @@ def user_list_params(counter, key_prefix)
         users.compact.uniq
       end
 
-      def project_timestamps
-        timestamps = {}
-        if project.metadata[:created_by].nil?
-          timestamps[:created_by] = current_user.uid
-          timestamps[:created_on] = Time.current.in_time_zone("America/New_York").iso8601
-
-        else
-          timestamps[:created_by] = project.metadata[:created_by]
-          timestamps[:created_on] = project.metadata[:created_on]
-          timestamps[:updated_by] = current_user.uid
-          timestamps[:updated_on] = Time.current.in_time_zone("America/New_York").iso8601
+      # Initializes values that we have defaults for.
+      def set_defaults
+        if @storage_capacity.nil?
```

Comment:
Using this version of the defaults will cause a regression. We need to be utilizing the Rails.configuration.project_defaults here

Reply:
Good catch. I've updated the code to read them from `Rails.configuration.project_defaults` again.